client/my-sites/site-settings/form-general.jsx

```diff
@@ -609,10 +617,26 @@ export class SiteSettingsFormGeneral extends Component {
 
 		const LaunchCard = showPreviewLink ? CompactCard : Card;
 
+		if ( isSiteOnECommerceTrial ) {
+			eCommerceTrialUpsell = (
+				<a href={ `/plans/${ siteSlug }?plan=business-bundle-monthly` }>
```

Comment:
Shouldn't this be the eCommerce bundle?

Reply:
I fixed it in my last commit. Thanks for noticing that :)